src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -85,19 +83,20 @@ public ICredentialProvider AcquireCredentials(Dictionary<String, String> args, b
             else
             {
                 // get from user
-                var consoleWizard = HostContext.GetService<IConsoleWizard>();
-                string authType = consoleWizard.ReadValue(CliArgs.Auth,
-                                                        StringUtil.Loc("AuthenticationType"),
-                                                        false,
-                                                        "PAT",
-                                                        Validators.AuthSchemeValidator,
-                                                        args,
-                                                        enforceSupplied);
+                var promptManager = HostContext.GetService<IPromptManager>();
+                string authType = promptManager.ReadValue(
```

Comment:
I have some other ideas how to simplify this as well. May be more appropriate as part of the back-compat args work.